cli/analytics.ts

```diff
@@ -1,21 +1,40 @@
+import yargs from "yargs";
+
 import Analytics from "analytics-node";
-import { getConfigSettings, getConfigSettingsPath, upsertConfigSettings } from "df/cli/config";
-import { ynQuestion } from "df/cli/console";
-import { v4 as uuidv4 } from "uuid";
+import {getConfigSettings, getConfigSettingsPath, upsertConfigSettings} from "df/cli/config";
+import {ynQuestion} from "df/cli/console";
+import {INamedOption} from "df/cli/yargswrapper";
+import {v4 as uuidv4} from "uuid";
+
+export const noTrackOption: INamedOption<yargs.Options> = {
```

Comment:
IMO rename to track, makes logic simpler. Otherwise `--no-track=false` is a double negative.

Also it shows that it overrides the default settings more clear.

Reply:
Renamed